templates/python/custom-copilot-rag-azure-ai-search/delete.py.tpl

```diff
@@ -0,0 +1,24 @@
+import os
+from azure.core.credentials import AzureKeyCredential
+from azure.search.documents.indexes import SearchIndexClient
+
+from dotenv import load_dotenv
+
+{{#enableTestToolByDefault}}
+load_dotenv(f'{os.getcwd()}/env/.env.testtool.user')
```

Comment:
please help confirm whether we should ask users put related envs to the `.env` file.

Reply:
let's follow the current environment management rules. I'll hold a feature discussion for this issue later.